lib/vsc/utils/fancylogger.py

```diff
@@ -345,6 +345,8 @@ def deprecated(self, msg, cur_ver, max_ver, depth=2, exception=None, *args, **kw
             self.raiseException("DEPRECATED (since v%s) functionality used: %s" % (max_ver, msg), exception=exception)
         else:
             deprecation_msg = "Deprecated functionality, will no longer work in v%s: %s" % (max_ver, msg)
+            if verbose:
+                sys.stderr.write(deprecation_msg)
```

Comment:
if you don't care about other warnigns or errors is in your logfiles, why do you care about deprecation warnings?